### PR TITLE
Feature/api

### PR DESCRIPTION
**Pull request by esrefbozkurt**
set up mongoDB, connect mongoDB, render exercises via entries State

### CODE REVIEW

File: components/Main/Exercises/Exercises.js [hunk 1]
```diff
@@ -6,7 +6,12 @@ import { useState } from "react";
 import SearchBar from "./SearchBar/SearchBar";
 import FavouriteButton from "./Favourite/FavouriteButton";
 
-export default function Exercises({ onFav, isFavourite, isFavouriteExercise }) {
+export default function Exercises({
+  onFav,
+  isFavourite,
+  isFavouriteExercise,
+  entries,
```
Comment (julialoeschel) [inline components/Main/Exercises/Exercises.js]: entires could be anything - exercises is most likely the better wording here.

File: pages/api/exercises/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          instructions: exercise.instructions,
+        };
+      });
+
+      res.status(200).json(exercisesArray);
+    } catch (error) {
+      console.log(error);
+      res.status(400).json(error);
+    }
+  }
```
Comment (julialoeschel) [inline pages/api/exercises/index.js:27]: cool!

**Review by julialoeschel — COMMENTED** [review]
Great job Eshi. 
Happy to see it is working 🐜 
I only have minor comments. 

keep up the good work!

File: components/Main/Exercises/Exercises.js (hunk 1 above)
Comment (esrefbozkurt) [reply components/Main/Exercises/Exercises.js]: makes sense, will change that

**Pull request merged**